pubsub/pulsar/pulsar.go

```diff
@@ -92,7 +94,22 @@ func parsePulsarMetadata(meta pubsub.Metadata) (*pulsarMetadata, error) {
 		}
 		m.DisableBatching = disableBatching
 	}
-
+	m.BatchingMaxMessages = 1000
```

Comment:
Please add TODO and document those hardcode.

Reply:
I looked at the pulasr client's API and can remove these hard coded code